### ISSUE TITLE
pages should be used consistently

### ISSUE THREAD

**Issue by thigg** [sailfishos-chum/bugger#3]
The welcome page does indicate a right action, without having one

it is quite uncommon that the primary action (submitting the form) is in the lower pulley (although it is nice because you went down already). As this is essentially a dialog, right page should send it.

**Comment by nephros** [comment]
The intention of the lower pulley was to make the user go through all the options.
If they need to go down, they are probably more likely to fill non-mandatory fields.

A side effect is possibly that they also won't send the form prematurely.

**Comment by thigg** [comment]
Yes, I understand that intention. It also appealed to me, because its very obvious once you get the idea.

Unfortunatley I know of no other app that does this and I think most users need to search for the submit action first.

**Comment by nephros** [comment]
0.9.5 fixes the first part.

Pullup-Menu thingie is planned for 0.9.6
